src/gui/dialogs/addon/manager.hpp

```diff
@@ -103,6 +103,7 @@ class addon_manager : public modal_dialog
 
 	static const std::vector<std::pair<ADDON_STATUS_FILTER, std::string>> status_filter_types_;
 	static const std::vector<std::pair<ADDON_TYPE, std::string>> type_filter_types_;
+	static std::vector<std::pair<int, std::string>> language_filter_types_;
```

Comment:
Since this is filled based on the languages of available addons, this should just be a regular class member and not static.

Reply:
:ok_hand: